diff --git a/virt/kvm/kvm_main.c b/virt/kvm/kvm_main.c
--- a/virt/kvm/kvm_main.c 2016-11-07 23:06:17.000000000 +0900
+++ b/virt/kvm/kvm_main.c 2016-11-20 13:38:34.987791311 +0900
@@ -74,7 +74,7 @@
 module_param(halt_poll_ns, uint, S_IRUGO | S_IWUSR);
 
 /* Default doubles per-vcpu halt_poll_ns. */
-static unsigned int halt_poll_ns_grow = 2;
+static unsigned int halt_poll_ns_grow = 0;
 module_param(halt_poll_ns_grow, uint, S_IRUGO | S_IWUSR);
 
 /* Default resets per-vcpu halt_poll_ns . */
